### PR TITLE
Docs for UC-6

### PR DESCRIPTION
**Pull request by jon-m-google**
# Description

Docs for UC-6 (Enforcing publisher ad requirements in PAAPI using K/V) incl:
- Main doc page
- Mermaid digram with link to large png version
- Changes to index.md + intro.md

## Affected services

- [ x] Home
- [ ] News
- [ ] Shop
- [ ] Travel
- [ ] DSP
- [ ] SSP
- [ ] ALL

### CODE REVIEW

File: services/home/docs/demos/publisher-ad-quality-req.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+</TabItem>
+<TabItem value="demo" label="Demo">
+
+## Demo
+
+### Prerequisites
+
+- Chrome > v128 (Open chrome://version to look up your current version)
+
```
Comment (Seburan) [inline services/home/docs/demos/publisher-ad-quality-req.md:133]: Suggest adding : 
- Enable Privacy Sandbox Protected Audience API (Open chrome://settings/adPrivacy and enable *Site-suggested ads*)

**Review by Seburan — CHANGES_REQUESTED** [review]

File: services/home/docs/demos/publisher-ad-quality-req.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+It’s in this part of the use case differs from the Retargeting / Remarketing & it uses a different news page.
+
+##### News page & SSP tag
+
+The publisher-ad-quality-req news page
+[(code link)](https://github.com/privacysandbox/privacy-sandbox-demos/blob/dev/services/news/src/views/publisher-ad-quality-req.ejs) reads the
+excludeCreativeTag and includes it in the sellerSignalExcludeCreativeTag as part of its adUnits list
+[(code link)](https://github.com/privacysandbox/privacy-sandbox-demos/blob/dev/services/news/src/views/publisher-ad-quality-req.ejs#L39). The news
+page also includes the SSP tag
+[(code link)](https://github.com/privacysandbox/privacy-sandbox-demos/blob/unified/services/ad-tech/src/public/js/ssp/ssp-tag.js) this will;
```
Comment (Seburan) [inline services/home/docs/demos/publisher-ad-quality-req.md:175]: use GitHub permalink feature instead which point to the [file](https://docs.github.com/en/repositories/working-with-files/using-files/getting-permanent-links-to-files) on a specific commit or [code](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-a-permanent-link-to-a-code-snippet). The unified branch will eventually be deleted but the commit will stay.

Example : 
https://github.com/privacysandbox/privacy-sandbox-demos/blob/1f9c6448072d922aa191f5ea2182ec92ead2fada/services/ad-tech/src/public/js/ssp/ssp-tag.js

**Review by Seburan — CHANGES_REQUESTED** [review]
please update permalinks to files and code snippets

File: services/home/docs/demos/publisher-ad-quality-req.md (hunk 1 above)
Comment (jon-m-google) [reply services/home/docs/demos/publisher-ad-quality-req.md:134]: Done

File: services/home/docs/demos/publisher-ad-quality-req.md (hunk 2 above)
Comment (jon-m-google) [reply services/home/docs/demos/publisher-ad-quality-req.md]: Done.

File: services/home/docs/demos/index.md [hunk 3]
```diff
@@ -8,12 +8,13 @@ sidebar_position: 1
...
+|         Show Relevant Video Ads          |   [Instream VAST video ad in a Protected Audience single-seller auction](demos/vast-video-protected-audience.md)   |             Protected Audience API             |      Publisher, SSP, Advertiser, DSP       |
+|      Show Relevant Content and Ads       |                           [Retargeting / Remarketing](demos/retargeting-remarketing.md)                            |             Protected Audience API             |      Publisher, SSP, Advertiser, DSP       |
+|           Measure Digital Ads            |                [Single-touch conversion Attribution](demos/single-touch-conversion-attribution.md)                 | Attribution Reporting API, Aggregation Service |      Publisher, SSP, Advertiser, DSP       |
+| Use SSP Key/Value service to exclude ads |            [Enforcing publisher ad requirements in PAAPI using K/V](demos/publisher-ad-quality-req.md)             |             Protected Audience API             |               Publisher, SSP               |
```
[the remark this answers is not in the document]
Comment (jon-m-google) [reply services/home/docs/demos/index.md]: Done

File: services/home/docs/demos/publisher-ad-quality-req.md [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+
+</TabItem>
+<TabItem value="scope" label="Scope">
+
+## Scope
+
+### Goals
+
+In this demo, we assume a publisher would like to exclude ads with certain product types. We’ll demonstrate a publisher page calling an SSP to perform
+a PAAPI auction, supplying product tags to exclude which have been selected by the user (e.g. “redShoe”). The SSP will exclude ads by calling their
```
[the remark this answers is not in the document]
Comment (jon-m-google) [reply services/home/docs/demos/publisher-ad-quality-req.md]: Done

File: services/home/docs/intro.md [hunk 5]
```diff
@@ -7,12 +7,13 @@ sidebar_position: 1
...
+|         Show Relevant Video Ads          |   [Instream VAST video ad in a Protected Audience single-seller auction](demos/vast-video-protected-audience.md)   |             Protected Audience API             |      Publisher, SSP, Advertiser, DSP       |
+|      Show Relevant Content and Ads       |                           [Retargeting / Remarketing](demos/retargeting-remarketing.md)                            |             Protected Audience API             |      Publisher, SSP, Advertiser, DSP       |
+|           Measure Digital Ads            |                [Single-touch conversion Attribution](demos/single-touch-conversion-attribution.md)                 | Attribution Reporting API, Aggregation Service |      Publisher, SSP, Advertiser, DSP       |
+| Use SSP Key/Value service to exclude ads |            [Enforcing publisher ad requirements in PAAPI using K/V](demos/publisher-ad-quality-req.md)             |             Protected Audience API             |               Publisher, SSP               |
```
[the remark this answers is not in the document]
Comment (jon-m-google) [reply services/home/docs/intro.md]: Done

File: services/home/docs/demos/publisher-ad-quality-req.md [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+### Key Exclusions
+
+- Dynamic updates of K/V pairs coming from the SSP backend (we will use static data for this demo).
+- Excluding ads based on DSP supplied meta-data.
+
+### System Design
+
+Using Protected Audience API, the user visits a shopping site, and gets added to an interest group. Later the same user visits a news site. Their
+browser runs an on-device auction & bidding logic will select the winning ad which will be shown to the user. The user now presses one of several
+buttons designed to exclude ads with certain product tags. The news page is refreshed, adding a url parameter for the excluded product tags. The new
```
[the remark this answers is not in the document]
Comment (jon-m-google) [reply services/home/docs/demos/publisher-ad-quality-req.md]: Done

**Review by Seburan — APPROVED** [review]

**Pull request merged**